lib/xbmcutil.py

```diff
@@ -218,11 +218,12 @@ def load_subtitles(url, headers=None):
         max_count = 99
         while not player.isPlaying() and count < max_count:
             count += 1
-            xbmc.sleep(200)
+            xbmc.sleep(2000)
```

Comment:
Isn't timeout of approximately 200 seconds too much ? Wouldn't be 30 seconds sufficient ?